picoboard/button/button.c

```diff
@@ -53,10 +53,14 @@ bool __no_inline_not_in_flash_func(get_bootsel_button)() {
 }
 
 int main() {
+#ifndef PICO_DEFAULT_LED_PIN
+#warning picobooard/button example requires a board with a regular LED
+#else
     gpio_init(PICO_DEFAULT_LED_PIN);
     gpio_set_dir(PICO_DEFAULT_LED_PIN, GPIO_OUT);
     while (true) {
-        gpio_put(PICO_DEFAULT_LED_PIN, get_bootsel_button());
+        gpio_put(PICO_DEFAULT_LED_PIN, get_bootsel_button() ^ PICO_DEFAULT_LED_PIN_INVERTED);
```

Comment:
Does this `PICO_DEFAULT_LED_PIN_INVERTED` also need to be used in all the other examples that use `PICO_DEFAULT_LED_PIN` ?

Reply:
i think most of the reset just flash it, so it doesn't much matter which way (at least that was my thinking)

Reply:
Good point. (assuming that the 'on' and 'off' periods are the same length, which they probably are!)